dash/testing/dash_page.py

```diff
@@ -36,13 +36,7 @@ def redux_state_paths(self):
     def redux_state_rqs(self):
         return self.driver.execute_script(
             """
-            return window.store.getState().pendingCallbacks.map(function(cb) {
-                var out = {};
-                for (var key in cb) {
-                    if (typeof cb[key] !== 'function') { out[key] = cb[key]; }
-                }
-                return out;
-            })
+            return !window.store.getState().isLoading;
```

Comment:
There's no `pendingCallbacks` anymore, changing usage for `isLoading`, which is equivalent to `|pendingCallbacks|!=0`

Reply:
This looks like it's inverting the logic - previously a truthy `redux_state_rqs` meant "we're waiting for something," now it's "we're NOT loading anything."

Given that this is part of our public `dash.testing` API, it seems like we should probably keep "no requests" as `[]`, though I don't think it matters (for backward compatibility) what we put in the array when there _are_ requests. Could we reduce it to a list of output strings? ie the `"..id.prop...id2.prop2.."` form for multi-output or `"id.prop"` for single - that could be useful for debugging failures.

There was also the question about maintaining backward compatibility with older versions of dash, in case someone wants to use this framework to test running apps that may include various dash versions, or perhaps for testing a version compatibility matrix. I already broke that for versions prior to 1.11 when I switched from `requestQueue` to `pendingCallbacks` - it would be nice to un-break that eventually, and in the meantime at least not break it again. Let's just check which of the relevant items exists in the store and return some list based on that, with the "nothing in progress" state being and empty list.